go/plugins/dotprompt/dotprompt.go

```diff
@@ -321,3 +329,99 @@ func sortSchemaSlices(s *jsonschema.Schema) {
 		sortSchemaSlices(s.Items)
 	}
 }
+
+// Adds a variant name to the prompt
+func WithVariant(variant string) PromptOption {
```

Comment:
This probably doesn't make much sense in code vs. files. I would leave it out for now.